pcmanfm/desktopwindow.cpp

```diff
@@ -948,7 +948,7 @@ bool DesktopWindow::eventFilter(QObject* watched, QEvent* event) {
             break;
         }
     }
-    return false;
+    return Fm::FolderView::eventFilter(watched, event);
```

Comment:
Are you skipping the direct parent's function intentionally?
(right, there is no override of `View::eventFilter()` now, but it can be added in the future...)